### PR TITLE
cloud: add a note for scaling 4C cluster

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/scale-tidb-cluster.md [hunk 1]
```diff
@@ -19,6 +19,14 @@ You can scale a TiDB cluster in the following dimensions:
 
 For information about how to determine the size of your TiDB cluster, see [Determine Your TiDB Size](/tidb-cloud/size-your-cluster.md).
 
+> **Note:**
+>
+> If the node size of TiDB or TiKV is set as **4 vCPU, 16 GiB**, note the following restrictions. To get rid of these restrictions, you can [increase your node size](#increase-node-size) first.
```
Comment (qiancai) [inline tidb-cloud/scale-tidb-cluster.md:24]:
```suggestion
> If the node size of TiDB or TiKV is set as **4 vCPU, 16 GiB**, note the following restrictions. To bypass these restrictions, you can [increase your node size](#increase-node-size) first.
```

**Review by SunRunAway — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
I am not sure whether we can use Latin expression.

File: tidb-cloud/scale-tidb-cluster.md [hunk 2]
```diff
@@ -19,6 +19,14 @@ You can scale a TiDB cluster in the following dimensions:
 
 For information about how to determine the size of your TiDB cluster, see [Determine Your TiDB Size](/tidb-cloud/size-your-cluster.md).
 
+> **Note:**
+>
+> If the node size of TiDB or TiKV is set as **4 vCPU, 16 GiB**, note the following restrictions. To bypass these restrictions, you can [increase your node size](#increase-node-size) first.
+>
+> - The node quantity of TiDB can only be set to 1 or 2, and the node quantity of TiKV is fixed to 3.
+> - 4 vCPU TiDB can only be used with 4 vCPU TiKV, and vice versa.
```
Comment (TomShawn) [inline tidb-cloud/scale-tidb-cluster.md:27]:
```suggestion
> - 4 vCPU TiDB can only be used with 4 vCPU TiKV, and 4 vCPU TiKV can only be used with 4 vCPU TiDB.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**